V2-alpha 3 exceptions in Command prompt

Operating System: Windows 10
JDK: openjdk 11
JDK used: https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11
![image](https://user-images.githubusercontent.com/37770/183638373-cdccba55-eefe-4b77-99fb-68621a88f412.png)

Let me know if there are any other system logs required.

@cleydyr I get the same exception as @cksachdev, here it is as text:
```
InteropFactory: cannot load com.sun.javafx.embed.swing.newimpl.InteropFactoryN
Graphics Device initialization failed for :  d3d, sw
Error initializing QuantumRenderer: no suitable pipeline found
java.lang.RuntimeException: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
        at com.sun.javafx.tk.quantum.QuantumRenderer.getInstance(QuantumRenderer.java:280)
        at com.sun.javafx.tk.quantum.QuantumToolkit.init(QuantumToolkit.java:222)
        at com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:260)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:267)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:158)
        at javafx.embed.swing.JFXPanel.initFx(JFXPanel.java:251)
        at javafx.embed.swing.JFXPanel.<init>(JFXPanel.java:267)
        at at.laborg.briss.BrissGUI.<init>(BrissGUI.java:123)
        at at.laborg.briss.Briss.main(Briss.java:34)
Caused by: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
        at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.init(QuantumRenderer.java:94)
        at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.run(QuantumRenderer.java:124)
        at java.base/java.lang.Thread.run(Unknown Source)
Exception in thread "main" java.lang.RuntimeException: No toolkit found
        at com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:272)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:267)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:158)
        at javafx.embed.swing.JFXPanel.initFx(JFXPanel.java:251)
        at javafx.embed.swing.JFXPanel.<init>(JFXPanel.java:267)
        at at.laborg.briss.BrissGUI.<init>(BrissGUI.java:123)
        at at.laborg.briss.Briss.main(Briss.java:34)
```

System configuration:
```
# systeminfo
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19045 N/A Build 19045
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
System Manufacturer:       LENOVO
System Model:              21A90007BM
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 141 Stepping 1 GenuineIntel ~2688 Mhz
BIOS Version:              LENOVO N38ET29W (1.10 ), 8/3/2021

# java -version
openjdk version "18.0.2" 2022-07-19
OpenJDK Runtime Environment Temurin-18.0.2+9 (build 18.0.2+9)
OpenJDK 64-Bit Server VM Temurin-18.0.2+9 (build 18.0.2+9, mixed mode, sharing)
```

--- a/build.gradle
+++ b/build.gradle
@@ -31,6 +31,9 @@ dependencies {
     runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:win"
     runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:linux"
     runtimeOnly "org.openjfx:javafx-swing:$javaFxVersion:mac"
+    runtimeOnly "org.openjfx:javafx-graphics:$javafx.version:win"
+    runtimeOnly "org.openjfx:javafx-graphics:$javafx.version:linux"
+    runtimeOnly "org.openjfx:javafx-graphics:$javafx.version:mac"
 
     testImplementation(platform('org.junit:junit-bom:5.9.1'))
     testImplementation('org.junit.jupiter:junit-jupiter')

